src/Drupal/Commands/core/MigrateRunnerCommands.php

```diff
@@ -57,64 +45,53 @@ class MigrateRunnerCommands extends DrushCommands
+        KeyValueFactoryInterface $keyValueFactory,
+        protected ?MigrationPluginManagerInterface $migrationPluginManager = null
+    ) {
         parent::__construct();
-        $this->dateFormatter = $dateFormatter;
         $this->keyValue = $keyValueFactory->get('migrate_last_imported');
-        $this->migrationPluginManager = $migrationPluginManager;
     }
 
     /**
-     * List all migrations with current status.
-     *
-     * @command migrate:status
-     *
-     * @param string|null $migrationIds
-     *   Restrict to a comma-separated list of migrations. Optional.
-     *
-     * @option tag A comma-separated list of migration tags to list. If only
-     *   <info>--tag</info> is provided, all tagged migrations will be listed,
-     *   grouped by tags.
-     * @option names-only [Deprecated, use --field=id instead] Only return names, not all the details (faster).
-     *
-     * @usage migrate:status
-     *   Retrieve status for all migrations
-     * @usage migrate:status --tag
-     *   Retrieve status for all migrations, grouped by tag
-     * @usage migrate:status --tag=user,main_content
-     *   Retrieve status for all migrations tagged with <info>user</info> or
-     *   <info>main_content</info>
-     * @usage migrate:status classification,article
-     *   Retrieve status for specific migrations
-     * @usage migrate:status --field=id
-     *   Retrieve a raw list of migration IDs.
-     * @usage ms --fields=id,status --format=json
-     *   Retrieve a Json serialized list of migrations, each item containing only the migration ID and its status.
-     *
-     * @aliases ms,migrate-status
-     *
-     * @topics docs:migrate
-     *
-     * @validate-module-enabled migrate
-     *
-     * @field-labels
-     *   id: Migration ID
-     *   status: Status
-     *   total: Total
-     *   imported: Imported
-     *   needing_update: Needing update
-     *   unprocessed: Unprocessed
-     *   last_imported: Last Imported
-     * @default-fields id,status,total,imported,unprocessed,last_imported
-     * @filter-default-field status
-     * @return RowsOfFields
-     *   Migrations status formatted as table.
-     * @version 10.4
+     * Lists all migrations with current status.
      *
      * @throws \Exception
      *   If --names-only is used with --field having other value than 'id'.
      */
+    #[CLI\Command(name: 'migrate:status', aliases: ['ms', 'migrate-status'])]
+    #[CLI\Argument(name: 'migrationIds', description: 'Restrict to a comma-separated list of migrations. Optional.')]
+    #[CLI\Option(name: 'tag', description: 'A comma-separated list of migration tags to list. If only <info>--tag</info> is provided, all tagged migrations will be listed, grouped by tags.')]
+    #[CLI\Option(name: 'names-only', description: '[Deprecated, use --field=id instead] Only return names, not all the details (faster).')]
+    #[CLI\Help(description: 'List all migrations with current status.')]
```

Comment:
You can omit this since annotated-command still uses the method description.

Reply:
OK, I did. But I don't necessarily agree. I think, now, attributes should be the norm. I would rather remove the method description.

Reply:
Yeah, I think that both approaches are valid. I think keeping the description in Doxygen is slightly more readable and @greg-1-anderson agrees since he made this work in https://github.com/consolidation/annotated-command/pull/281.